### ISSUE TITLE
When clicking the menu, the menu will "blink"

### ISSUE THREAD

**Issue by abedputra** [jeroennoten/Laravel-AdminLTE#763]
I want to report when I implement your template, I saw something wrong.

**Describe the bug**
I saw the menu when clicking the menu, the menu will "blink", Please check the gif

**To Reproduce**
when clicking the navbar

**Screenshots**
![blink-navbar](https://user-images.githubusercontent.com/11581453/98831770-f16b1600-2476-11eb-8cc6-a92532747983.gif)

**Environment**
- Operating System:  [macOS Catalina]
- Browser (Version): [Chrome (Latest), also Safari]
- Laravel: 7
- Laravel-AdminLTE : 3.5.X

Thank you

**Comment by Shidersz** [comment]
@abedputra The menu is refreshed (reload) on every request, because the active item of the menu is recalculated every time based on the current url. Anyway, I have also noted a extra blink effect on it. I have digger into this and found that including next plugin in the `master.blade.php`  is causing such issue (at least on my environment):

```blade
<script src="{{ asset('vendor/overlayScrollbars/js/jquery.overlayScrollbars.min.js') }}"></script>
```

Can you try to comment it like next and check if it get better?

```blade
{{-- <script src="{{ asset('vendor/overlayScrollbars/js/jquery.overlayScrollbars.min.js') }}"></script> --}}
```

If you have [published](https://github.com/jeroennoten/Laravel-AdminLTE/wiki/11.-Customize-Views) the **main views** you need to edit the file:

```sh
resources/views/vendor/adminlte/master.blade.php
```

Otherwise, you will found it on:

```sh
vendor/jeroennoten/laravel-adminlte/resources/views/master.blade.php
```

Note that the plugin is used to generate a nice scroll on the sidebar. But knowing it is generating the issue I will have an starting point to go deeper.

**Comment by abedputra** [comment]
Hi, @Shidersz Thanks for your reply.
Yes, I know when the reload page will be recalculated. But when I compare with the demo of AdminLTE my own more blink.
Please check it.
![Screen-Recording-2020-11-12-at-0](https://user-images.githubusercontent.com/11581453/98881257-88aa8a80-24c4-11eb-80c6-a7d2b71bd20a.gif)

I tried commenting like you said on the "overlayScrollbars", but still blink.
![Screen Shot 2020-11-12 at 08 55 10](https://user-images.githubusercontent.com/11581453/98881409-e212b980-24c4-11eb-8abf-767a85187c4b.png)

**Comment by abedputra** [comment]
You can check also check the blink here, from my web
http://laravel.connectwithdev.com/login
u: administrator@gmail.com
p: 123456789

Thank you

**Comment by Shidersz** [comment]
> Yes, I know when the reload page will be recalculated. But when I compare with the demo of AdminLTE my own more blink.
Please check it.

@abedputra It is hard to compare because this project is build on top of Laravel (that uses blade engine, etc...) while the demo site is just `HTML`, `Javascript` and `CSS`. However, I will analyze if this can be enhanced on my free time. Like I said, on my local environment, commenting out the `overlayScrollbars.min.js` prevent multiple blinks (but I still get one blink on every reload).

**Comment by abedputra** [comment]
Thanks a lot, @Shidersz.

**Comment by REJack** [comment]
You can also disable the layout "fixed" mode, that should ignore overlayScrollbars. Only the fixed mode needs overlayScrollbars.

**Comment by Shidersz** [comment]
@REJack I have found the source of the problem, it is not **overlayScrollbars**, the blink is caused by animations added by next styles from **_variables.scss:36** (that are inserted on `<p>` tags of the sidebar links):

```css
.nav-sidebar .nav-link p {
    display: inline-block;
    -webkit-animation-name: fadeIn;
    animation-name: fadeIn;
    -webkit-animation-duration: .3s;
    animation-duration: .3s;
    -webkit-animation-fill-mode: both;
    animation-fill-mode: both;
    margin: 0;
}
```

However, on the [AdminLTE v3 demo site](https://adminlte.io/themes/v3/) the styles applied to the `<p>` tags are different and comes from **_main-sidebar.scss:69**:

```css
.nav-sidebar .nav-link p {
    display: inline-block;
    margin: 0;
}
```

I can use a workaround to solve this by adding `style="animation:none;"` to the `<p>` tags, but I believe the solution should be other, any idea why the styles applied are not the same. The `master.blade.php` file just includes the `<link rel="stylesheet" href="{{ asset('vendor/adminlte/dist/css/adminlte.min.css') }}">` as on the demo site.

**Comment by Shidersz** [comment]
@abedputra You can use next solution temporarily. Locate and edit the package file `page.blade.php` and replace next piece of code:

```blade
@section('adminlte_css')
    @stack('css')
    @yield('css')
@stop
```

by next code:

```blade
@section('adminlte_css')
    <style type="text/css">
        .nav-sidebar .nav-link p {
            animation:none;
        }
    </style>
    @stack('css')
    @yield('css')
@stop
```
That should remove the animations that generates the blink effect. @REJack I think this problem should be solved on the `AdminLTE` package.

**Comment by abedputra** [comment]
@Shidersz Thank you... Let me check it and implement it.

Thanks

**Comment by abedputra** [comment]
@Shidersz your CSS is great! it will remove the blink.

thanks

**Comment by resslinger** [comment]
@REJack

**Comment by REJack** [comment]
This will be fixed with AdminLTE v3.1.0.
